index.html

```diff
@@ -0,0 +1,189 @@
+    <!-- /Stylesheets -->
+</head>
+
+<body>
+    <!-- Navigation Bar -->
+    <nav class="navbar navbar-default navbar-fixed-top" role="navigation">
+        <div class="container">
+            <a class="navbar-brand" href="#">MATKA</a>
+            <ul class="nav navbar-nav">
+                <li>
+                    <a href="#">Q'17</a>
+                </li>
+                <li>
+                    <a href="#">Sponsors</a>
+                </li>
+                <!--                 <li>
+                    <a href="#">Contacts</a>
+                </li> -->
+                <li>
+                    <a href="#">Register</a>
+                </li>
+            </ul>
+        </div>
+    </nav>
+    <!-- /Navigation Bar -->
+    <!-- Section 1 -->
+    <section class="section1">
+        <div class="s1imgwrapper">
+            <div class="qlogo-container">
+                <img src="public/img/quark_black_composite2.png" alt="" id="qlogo">
+            </div>
+            <div class="bpgclogo-container">
+                <img src="public/img/bpgclogo.png" alt="" id="bpgclogo">
+            </div>
+            <div class="mlogo-container"><img src="public/img/matka_logo.png" alt="" id="mlogo"></div>
+            <div id="prize">
+                <span><p>Prize pool worth</p><p>₹ 3 Lacs!</p></span>
+                <span><p>Online Qualifiers from</p><p>10 to 15th January!</p></span>
+            </div>
+            <div class="aboutus">
+                <p>
+                    <b>Quark 2017</b> is proud to present the Biggest College Gaming Event in India - <b>MATKA</b>, for games like <b>CS:GO</b> and <b>DOTA 2</b> with a prize pool worth
+                    <br><b id="prizemoney">₹ 3 Lacs</b>. We are proud to associate with <b class="sponsortext">SoStronk</b>, <b class="sponsortext">BenQ</b>, <b class="sponsortext">Gaming Central</b> and <b class="sponsortext">Source</b> as our event partners. The event consists of online qualifiers for CS-GO and DOTA 2 scheduled from <b>10th -15th January 2017</b>. CS- GO qualifiers will be hosted on SoStronk server while DOTA 2 will be hosted on source servers. The top <b>8</b> teams for each game will qualify for the LAN finals, <b>4</b> more teams will join them in the finals selected through Goan Qualifiers (Exclusively for Goans). The dates for the LAN finals are <b>3rd to 5th February 2017</b> and will happen during Quark 2017 at BITS PILANI K.K. Birla Goa Campus. <a id="registration-link">Register here</a> to participate in online qualifiers for CS-GO and DOTA 2. Registration is <b>FREE</b> for everyone. Last date for registrations is <b>5th January 2017!</b>
+                </p>
+            </div>
+        </div>
+    </section>
+    <section class="section2">
+        <div class="left-half grid">
+        </div>
+        <div class="right-half grid">
+            <div class="container-fluid">
+                <img src="public/img/cslogo2.png" alt="" id="csgologo">

```

Comment:
Remove public from the urls. So it will be just `<img src="img/cslogo2.png" alt="" id="csgologo">`